### PR TITLE
macOS: Update build instructions 

### PR DESCRIPTION
**Pull request by JonasBorchelt**
Let it build

### CODE REVIEW

File: copying.md [hunk 1]
```diff
@@ -6,139 +6,139 @@ A copy of the license can be found in [legal/GPLV3](/legal/GPLv3).
...
+| Prashanth Jonnala           | jprashanth                  | prashanth dawt neo à gmail dawt com              |
+| Jonathan Remnant            | Jon0                        | jono4728 à gmail dawt com                        |
+| Sam Schetterer              | schets                      | samschet à gmail dawt com                        |
+| Georg Kilzer                | leper                       | leper à wildfiregames dawt com                   |
+| Florian Erler               | ethon                       | ethon à ethon dawt cc                            |
+| Michał Janiszewski          | janisozaur                  | janisozaur+openage à gmail dawt com              |
+| Lautaro Nahuel De León      | lndl                        | laudleon à gmail dawt com                        |
+| Robin Kreis                 | rkreis                      | r dawt kreis à uni-bremen dawt de                |
+| Shion Ryuu                  | shion                       | shionryuu à outlook dawt com                     |
+| Jonas Borchelt              | riotjones                   | jonas dawt borchelt à connected dawt link        |
```
Comment (heinezen) [inline copying.md:48]: This will make kevin complain about the old commits. You should add your new mail to `.mailmap` instead.

**Review by heinezen — COMMENTED** [review]
Thanks a lot. Could you also fix the macOS CI workflow as a "proof-of-concept"? I can't test the fixes locally because I don't have Apple hardware :D

**Comment by duanqn** [comment]
> Thanks a lot. Could you also fix the macOS CI workflow as a "proof-of-concept"? I can't test the fixes locally because I don't have Apple hardware :D

@heinezen
Verified by picking the CMake change. MacOS CI succeeded ~(with overwhelming warnings)~!
https://github.com/duanqn/openage/runs/4535817928?check_suite_focus=true

**Comment by heinezen** [comment]
@duanqn Nice one. I transferred your changes to this PR.

File: doc/building.md [hunk 2]
```diff
@@ -116,6 +116,13 @@ Make sure you have all the dependencies installed.
 - Select the compiler and mode: see `./configure --help`
   - Linux etc: `./configure`
   - macOS:  `./configure --compiler=clang`
+    - hassle-free building on macOS (if above is not working)
+      - get the latest llvm build from: `https://github.com/llvm/llvm-project/releases/latest`
+        - (yeah, xcode still needs to be installed)
+      - install python deps: see [pip(3) install](https://github.com/SFTtech/openage/tree/master/doc/build_instructions/macos.md)
+      - install build deps (without llvm): see [brew install](https://github.com/SFTtech/openage/tree/master/doc/build_instructions/macos.md)
+      - configure: `CC=/path/to/downloaded-llvm/bin/clang CXX=/path/to/downloaded-llvm/bin/clang++ LD=/path/to/downloaded-llvm/bin/ld64.lld ./configure --download-nyan`
+      - a smoll build hint for later: use your freaking cores with the -j option later `make -j6`
```
[the remark this answers is not in the document]
Comment (duanqn) [reply doc/building.md:125]: smoll -> small ?

[the remark this answers is not in the document]
Comment (JonasBorchelt) [reply doc/building.md]: good hint @heinezen 
@duanqn smoll was intentional, but I've changed it.

File: copying.md (hunk 1 above)
Comment (JonasBorchelt) [reply copying.md]: check

**Review by heinezen — APPROVED** [review]
Good job, guys.

**Pull request merged**